staging/src/k8s.io/apiextensions-apiserver/pkg/cmd/server/options/options.go

```diff
@@ -127,16 +106,3 @@ func NewCRDRESTOptionsGetter(etcdOptions genericoptions.EtcdOptions) genericregi
 
 	return ret
 }
-
-func (o CustomResourceDefinitionsServerOptions) RunCustomResourceDefinitionsServer(stopCh <-chan struct{}) error {
```

Comment:
just curious, why remove this from here and add in `server.go`?

Reply:
RunCustomResourceDefinitionsServer exists in options.go is odd.